t/25-downloader.t

```diff
@@ -68,28 +68,30 @@ sub stop_server {
 }
 
 my $mojo_tmpdir = tempdir;
-my $downloader  = OpenQA::Downloader->new(log => $log, sleep_time => 1, tmpdir => $mojo_tmpdir);
+my $downloader  = OpenQA::Downloader->new(log => $log, sleep_time => 0.05, attempts => 3, tmpdir => $mojo_tmpdir);
+my $ua          = $downloader->ua;
 my $tempdir     = tempdir;
 my $to          = $tempdir->child('test.qcow');
 
 subtest 'Connection refused' => sub {
     my $from = "http://$host/tests/922756/asset/hdd/sle-12-SP3-x86_64-0368-textmode@64bit.qcow2";
+    $ua->connect_timeout(0)->inactivity_timeout(0);
```

Comment:
Why do you want to disable the timeouts completely here? I thought the goal was to reduce the runtime.

Reply:
Is the timeout even relevant in this test? We probably don't expect arbitrary overloads etc. in this context?

Reply:
I thought zero would reduce the timeout to zero but ok, if it disables the timeout that's bad of course.

Reply:
So that's a "Yes"? I can't reproduce any failures and I've not seen it fail on CircleCI so far.

Reply:
The request is supposed to fail here as 'Connection refused' anyways. I assumed setting a timeout of zero would be beneficial so it is treated as failure immediately. But it seems like that's happening anyways and also with infinite timeout (which the 0 actually stands for).

Reply:
#3531